compiler-core/src/error.rs

```diff
@@ -1032,20 +1032,67 @@ https://gleam.run/getting-started/installing/",
+                    "deno" => text.push_str(
                         "
-You can also install Erlang via homebrew using \"brew install erlang\"",
+Documentation for installing Deno can be viewed here:
+https://docs.deno.com/runtime/getting_started/installation/",
                     ),
-                    ("rebar3", "macos") => text.push_str(
+                    "node" => text.push_str(
                         "
-You can also install rebar3 via homebrew using \"brew install rebar3\"",
+Documentation for installing Node.js via package manager can be viewed here:
+https://nodejs.org/en/download/package-manager/all/",
                     ),
                     _ => (),
-                };
+                }
+                match env::consts::OS {
+                    "macos" => {
+                        fn brew_install(program: &str) -> String {
+                            format!("\nYou can install {} via homebrew: brew install {}", program, program)
+                        }
+                        match program.as_str() {
+                            "erl" | "erlc" | "escript" => text.push_str(&brew_install("erlang")),
+                            "rebar3" => text.push_str(&brew_install("rebar3")),
+                            "deno" => text.push_str(&brew_install("deno")),
+                            "elixir" => text.push_str(&brew_install("elixir")),
+                            other => text.push_str(&wrap_format!("
+You might need to install {other} manually on your macOS."
+                            )),
+                        }
+                    }
+                    "linux" => {
+                        fn apt_install(program: &str) -> String {
+                            format!("\nYou can install {} via apt: sudo apt install {}", program, program)
+                        }
+                        let release = std::fs::read_to_string("/etc/os-release").ok();
+                        if let Some(release) = release {
+                            if release.contains("ubuntu") || release.contains("debian") {
+                                match program.as_str() {
+                                    "erl" | "erlc" | "escript" => text.push_str(&apt_install("erlang")),
+                                    "elixir" => text.push_str(&apt_install("elixir")),
+                                    other => text.push_str(&wrap_format!("
+You might need to install {other} manually on your Linux distribution.
+Please refer to the software's documentation for installation instructions."
+                                    )),
+                                }
+                            } else {
+                                text.push_str(&wrap_format!("
+You might need to install {program} manually on your Linux distribution.
+Please refer to the software's documentation for installation instructions."
+                                ));
+                            }
+                        }
+                    }
```

Comment:
I think this text without any instructions is just noise- the error already suggests that installation is required at the start.

Reply:
removed : )